README.md

```diff
@@ -4,7 +4,7 @@ Analyze FPGA tool performance (MHz, resources, runtime, etc)
 
 ## Setup environment
 
-FPGA tool perf uses the Anaconda (conda) package manager to install and get all the required tools.
+fpga-tool-perf uses the Anaconda/Miniconda (conda) package manager to install and get all the required tools.
```

Comment:
I know fpga-tool-perf uses Miniconda, which is not quite the same as Anaconda. Would it be better to say just `Miniconda (conda)`?

Reply:
So, Miniconda is the light version of Anaconda, but to avoid confusion yes, I think that only `Miniconda (conda)` would is better